src/App.js

```diff
@@ -1,41 +1,17 @@
 import "./App.css";
-import logo from "./logo.png";
+import { Routes, Route } from "react-router-dom";
+import { LandingPage } from './Pages';
+import { Home} from "./Pages/index";
+import { Navbar } from "./component/index";
```

Comment:
You have added the config for path alias so you can omit `./` from the imports

Reply:
Actually I added config path after these imports and forgot to update it later. Will change it now.